core/src/Authorization.html

```diff
@@ -159,17 +181,32 @@
           const logoutItem = await LuigiConfig.getConfigValueAsync(
             'navigation.profile.logout'
           );
+          //check if the User Settings schema exist
+          const userSettings = await LuigiConfig.getConfigValueAsync(
+            'settings.userSettings'
+          );
+          isUserSettings = Boolean(userSettings);
```

Comment:
I felt `hasUserSettings` more understandable. Otherwise, it should be `isUserSettingsExisted`. What do you think?

Reply:
Changed it into hasUserSettings.